packages/fx-core/src/common/samples-config.json

```diff
@@ -205,6 +205,18 @@
             "time": "5min to run",
             "configuration": "Ready for local debug",
             "suggested": false
+        },
+        {
+            "id": "query-org-user-with-message-extension-sso",
+            "title": "Query with Message Extension SSO",
+            "shortDescription": "Query Org User with Message Extension SSO",
+            "fullDescription": "This is a simple search-based message extension app demonstrating how to integrate with Bot Framework SDK, TeamsFx SDK and Microsoft Graph API to implement a feature that queries organization's user with single sign on (SSO) capability.",
+            "tags": ["Message extension", "SSO"],
+            "time": "5min to run",
+            "configuration": "Ready for local debug",
+            "suggested": false,
+            "url": "https://github.com/OfficeDev/TeamsFx-samples/tree/dev/",
```

Comment:
will remove the url and packageLink property after v1.0.0 tag contains this sample